src/ML/src/MANNAutoregressive.cpp

```diff
@@ -607,10 +664,11 @@ bool MANNAutoregressive::advance()
     // we update the projected contact position in the world frame if and only if
     // 1. The support foot changed
     // 2. Tue support foot did not change but the lower corner yes
-    if ((supportFootPtr != m_pimpl->supportFootPtr)
-        || (supportFootPtr == m_pimpl->supportFootPtr
-            && supportCornerIndex != m_pimpl->supportCornerIndex))
+    if ((supportFootPtr != m_pimpl->supportFootPtr))
+    // || (supportFootPtr == m_pimpl->supportFootPtr
+    //     && supportCornerIndex != m_pimpl->supportCornerIndex))
```

Comment:
Commented code

Reply:
Fixed in 7461bf0f7. Moreover, I added a comment that explains the rationale behind the if statement.